app/models/api_commercial_search.rb

```diff
@@ -42,42 +38,4 @@ def instantiate_engine(_options)
   def domains_scope_options
     DomainScopeOptionsBuilder.build(site: @affiliate, site_limits: nil)
   end
-
-  def handle_response(response)
-    return unless response
-    @results = response.results
-    @next_offset = response.next_offset
-    true
-  end
-
-  def populate_additional_results
-    @govbox_set = GovboxSet.new(@query,
-                                @affiliate,
-                                nil,
-                                @highlight_options) if first_page?
-  end
-
-  def first_page?
-    @offset.zero?
-  end
-
-  def as_json_build_snippet(description)
-    description
-  end
-
-  def as_json_append_govbox_set(hash)
-    super do
-      hash[:recent_video_news] = video_news_items ? as_json_video_news(video_news_items.results) : []
-      hash[:recent_news] = news_items ? as_json_recent_news : []
-    end
-  end
-
-  def as_json_recent_news
-    news_items.results.map { |news_item| as_json_news_item news_item }
-  end
-
-  def log_serp_impressions
-    @modules << default_module_tag if @results.present?
-    @modules |= @govbox_set.modules if @govbox_set
-  end
```

Comment:
Please back out the changes to this file. Without doing a code crawl, I suspect, considering that we still have passing specs, that either:
- the methods are indeed not in use, but I'm nervous about tearing out methods from a parent class before we start making a lot of API changes
OR
- we're actually missing some specs that might fail without these methods.

Our APIs aren't fully covered by integration specs, so I'm not fully confident that removing them wouldn't break anything. Let's be safe and revert the changes in this file.